draft-dcook-ppm-dap-interop-test-design.md

```diff
@@ -128,10 +121,10 @@ responses for each endpoint listed below SHALL be encoded JSON objects
 {{!RFC8729}}, with media type `application/json`. All binary blobs (i.e. task
 IDs, batch IDs, HPKE configurations, and verification keys) SHALL be encoded as
 strings with base64url {{!RFC4648}}, inside the JSON objects. Any integer values
-in a VDAF's parameters, measurement, or aggregate result will be encoded as
-strings in base 10 instead of as numbers. This avoids incompatibilities due to
-limitations on the range of JSON numbers that different implementations can
-process.
+in the parameters, measurement, or aggregate result of a {{VDAF}} will be
```

Comment:
If you change this to {{!VDAF}}, then `make` will automatically render the normative reference for you.

Reply:
It's already normative due to its position in the YAML document up top (which also sets up the "VDAF" symbol). See [here](https://divergentdave.github.io/draft-dcook-ppm-dap-interop-test-design/david/vdaf-reference/draft-dcook-ppm-dap-interop-test-design.html#name-normative-references) for the output.